delocate/libsana.py

```diff
@@ -24,6 +31,9 @@ def tree_libs(start_path, filt_func=None):
         If None, inspect all files for library dependencies. If callable,
         accepts filename as argument, returns True if we should inspect the
         file, False otherwise.
+    ignore_missing : bool
```

Comment:
This should be:

```
ignore_missing : {False, True}, optional
```

Reply:
Sorry for that.  Right now I'm more used to MyPy type-hinting where it's normally bool, and optional has a different meaning.